src/rdbfmsua.fd/CMakeLists.txt

```diff
@@ -16,5 +14,6 @@ target_link_libraries(${exe_name} PRIVATE bufr::bufr_4
                                           w3emc::w3emc_4
                                           gempak::gempak
                                           gfortran::gfortran)
+#target_link_libraries(${exe_name} PRIVATE "-lgfortran")
```

Comment:
I kept it because that also worked.  With that line, one does not need to use `Findgfortran.cmake`, which means it picks up the one that is in the LD_LIBRARY_PATH (presumably the right one).  I retained it to not get lost after squashing our commits.